packages/api/src/controllers/IdentitiesController.js

```diff
@@ -14,7 +24,9 @@ class IdentitiesController {
       return response.status(404).send({ message: 'not found' })
     }
 
-    response.send(identity)
```

Comment:
Remove also balance count from the DAO

Reply:
was removed from getIdentityByIdentifier, but remained in getIdentities because otherwise we can't filter by balance